src/systems/joint_position_controller/JointPositionController.cc

```diff
@@ -240,14 +258,56 @@ void JointPositionController::PreUpdate(
     return;
   }
 
-  // Update force command.
+  // Get error in position
   double error;
   {
     std::lock_guard<std::mutex> lock(this->dataPtr->jointCmdMutex);
     error = jointPosComp->Data().at(this->dataPtr->jointIndex) -
             this->dataPtr->jointPosCmd;
   }
 
+  // Check if the mode is ABSOLUTE
+  if (this->dataPtr->mode ==
+    JointPositionControllerPrivate::OperationMode::ABSOLUTE)
+  {
+    // Calculate target velcity
+    double targetVel = 0;
+
+    // Get time in seconds
+    auto dt = std::chrono::duration<double>(_info.dt).count();
+
+    // Get the maximum amount in m that this joint may move
+    auto maxMovement = this->dataPtr->posPid.CmdMax() * dt;
```

Comment:
Should we also use `CmdMin` for negative velocities?

Reply:
I don't think its absolutely essential. We could add this in a future PR/release. I guess it would simulate actuators which have asymmetric forward and backward speeds. That being said, I do not feel it is necessary immediately. To be honest, this mode will be used usually when we don't really care about motor characteristics.